content/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization.md

```diff
@@ -32,7 +32,7 @@ Organization owners have full administrative access to the organization. {% data
 {% endnote %}
 
 {% ifversion enterprise-owner-join-org %}
-If your organization is owned by an enterprise account, any enterprise owner can make themself an owner of your organization. For more information, see "[AUTOTITLE](/admin/user-management/managing-organizations-in-your-enterprise/managing-your-role-in-an-organization-owned-by-your-enterprise)."
+If your organization is owned by an enterprise account, any enterprise owner can make themselves an owner of your organization. For more information, see "[AUTOTITLE](/admin/user-management/managing-organizations-in-your-enterprise/managing-your-role-in-an-organization-owned-by-your-enterprise)."
```

Comment:
#### Before
https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization#about-maintaining-ownership-continuity-for-your-organization
<img width="736" alt="image" src="https://github.com/github/docs/assets/2119212/55d83497-33d3-4010-8b0f-ab44d67b8c0c">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/enterprise-cloud@latest/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization#about-maintaining-ownership-continuity-for-your-organization
<img width="725" alt="image" src="https://github.com/github/docs/assets/2119212/e9a5d96d-3f34-4757-ad06-3926f0f929b0">